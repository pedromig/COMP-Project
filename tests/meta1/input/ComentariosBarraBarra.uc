x//>
//(
+aaaa
//boas
ola//ss