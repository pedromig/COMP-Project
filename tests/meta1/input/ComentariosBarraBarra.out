ID(x)
PLUS
ID(aaaa)
ID(ola)
